src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj

```diff
@@ -101,6 +102,7 @@
     <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\ECDsa\ECDsaTestData.cs">
       <Link>CommonTest\System\Security\Cryptography\AlgorithmImplementations\ECDsa\ECDsaTestData.cs</Link>
     </Compile>
+    <Compile Include="RSACngProvider.cs" />
```

Comment:
Hard-ordered. The previous ordering however had a mix of both ways plus some randomness... So I think I just made them alphabetical by file name for consistency. I really didn't have a preference either way (alphabetical vs location+alphabetical)